cmd/lotus-miner/info.go

```diff
@@ -95,284 +102,289 @@ func infoCmdAct(cctx *cli.Context) error {
 
 	fmt.Println()
 
-	maddr, err := getActorAddress(ctx, cctx)
-	if err != nil {
-		return err
-	}
-
-	mact, err := api.StateGetActor(ctx, maddr, types.EmptyTSK)
-	if err != nil {
-		return err
-	}
-
-	tbs := blockstore.NewTieredBstore(blockstore.NewAPIBlockstore(api), blockstore.NewMemory())
-	mas, err := miner.Load(adt.WrapStore(ctx, cbor.NewCborStore(tbs)), mact)
-	if err != nil {
-		return err
-	}
+	if subsystems.Has(api.SectorStorageSubsystem) {
```

Comment:
@raulk this is why I prefer to have the API typed, and not `[]string` - it is serialised as []string already.